src/app/utils/utils.js

```diff
@@ -524,6 +525,18 @@ function encodeHTML(str) {
   return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
 }
 
+/**
+ * extractNoticePreference(fixedFields)
+ * Returns 'None', 'Email', or 'Telephone'.
+ * @param {string} fixedFields - Object coming from patron object `fixedFields` property
+ */
+function extractNoticePreference(fixedFields) {
+  if (!fixedFields) return 'None';
+  const noticePreferenceField = fixedFields['268'];
+  if (!noticePreferenceField || !noticePreferenceField.value) return 'None';
+  return noticePreferenceMapping[noticePreferenceField.value];
```

Comment:
Should you check for the case where the key doesn't exist in `noticePreferenceMapping` (e.g. they add a code for SMS or postal mail or..). Currently, if you encounter a code that doesn't exist it will just render as `undefined` / blank, which may be fine..

Reply:
I'd say `None` is a better default than `undefined`